tests/functional/test_cli.py

```diff
@@ -53,11 +53,13 @@ def test_run_hello_workflow(self):
                 "artifacts_dir": self.artifacts_dir,
                 "scratch_dir": "/ignored",
                 "manifest_path": "/ignored",
+                "runtime": None,
```

Comment:
I had to explicitly set this to None, matching it to my python runtime didnt work, and I'm not sure why. Integ tests still work.

Reply:
We should set `runtime` to something like `ignored` so the validator fails open.

Reply:
👍